cpp/models/lct_secir/model.h

```diff
@@ -88,7 +280,6 @@ class Model
     }
 
     Parameters parameters{}; ///< Parameters of the model.
-    InfectionState infectionState; ///< InfectionState specifies number of subcompartments.
 
 private:
     Eigen::VectorXd m_initial_values; ///< Initial values of the model.
```

Comment:
I think this topic should be a part of the issue #914. I will add the idea there and rewrite the issue because of the changed conditions due to this PR.

Reply:
Sounds good 👍